handlers/object-id.js

```diff
@@ -27,11 +29,15 @@ module.exports = function (elastic, config) {
       jsonResult.found = true;
       jsonResult.error = null;
       jsonResult.searchError = error;
-      jsonResult.path = '/objects/' + result.hits.hits[0]._id;
+
+      var obj = result.hits.hits[0];
+      var slugValue = obj._source.summary_title && slug(obj._source.summary_title).toLowerCase();
```

Comment:
@jamieu this looks good _now_ but it would be good to either have a comment or a comment linking to the issue for future ref. 👍